app.rb

```diff
@@ -78,6 +78,11 @@ class App < Sinatra::Base
       return ""
     end
 
+    if sender_is_repetitive?(source, message)
+      logger.warn("Possible bot-number detected: #{source}")
```

Comment:
Maybe just say here that we received too many messages instead? At a later date when we've forgotten about the incident, we may not understand this